tasks/create_record.rb

```diff
@@ -0,0 +1,31 @@
+#!/opt/puppetlabs/puppet/bin/ruby
+
+require_relative '../../ruby_task_helper/files/task_helper.rb'
+require '../lib/service_now_request.rb'
+
+# This task creates records
+class ServiceNowCreate < TaskHelper
+  def task(table: nil,
+           fields: nil,
+           _target: nil,
+           **_kwargs)
+
+    user = _target[:user]
+    password = _target[:password]
+    instance = _target[:name]
+
+    uri = "https://#{instance}.service-now.com/api/now/table/#{table}"
+
+    begin
+      request = ServiceNowRequest.new(uri, 'Post', fields, user, password)
+      request.print_response
+    rescue => e
```

Comment:
Since the only possible errors are from the request, I think the `begin/end` block could move over to `print_response`. Then you can have in `print_response`
```
begin
rescue => e
  raise TaskHelper::Error...
end
```

Note that a Ruby exception will always print to stderr, and that we will print the right thing because `TaskHelper::Error` handles that. So, having `STDERR.puts` is redundant.

Reply:
Standardized the error handling. Removed redundant STDERR.puts.

Reply:
moved begin block to ServiceNowRequest.rb